### PR TITLE
Update exclude section in .flake8

### PR DESCRIPTION
**Pull request by ancorgs**
As far as I can see, there is no longer any reason to exclude `openqa-maintenance.py` and `openqa` is not longer there.

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -1,4 +1,4 @@
 [flake8]
-exclude = .open-build-service/, abichecker, openqa, openqa-maintenance.py
+exclude = .open-build-service/, abichecker
```
Comment (coolo) [inline .flake8:2]: .open-build-service isn't here either. Basically all of 495878ff0f460b7f44dded36b8600ba8fdd8a794 can be reverted

Comment (ancorgs) [reply .flake8]: Done. Added a new commit for that.

**Comment by coolo** [comment]
the openSUSE infrastructure had a bad weekend :(

**Review by coolo — APPROVED** [review]

**Comment by ancorgs** [comment]
Since we are not in a hurry, I will wait until the openSUSE infrastructure is awake to re-run the tests and merge in green.

**Pull request merged**